### PR TITLE
feat(menu): add animations

### PR DESCRIPTION
**Pull request by kara**
This PR adds animations (including the ripple) to the menu.

r: @hansl

### CODE REVIEW

File: src/lib/menu/menu-item.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<ng-content></ng-content>
+<div class="md-menu-ripple" *ngIf="!disabled" md-ripple md-ripple-background-color="rgba(0,0,0,0)"
+     [md-ripple-trigger]="_getHostElement()">
+</div>
```
Comment (hansl) [inline src/lib/menu/menu-item.html]: EOL.

File: src/lib/menu/menu.scss [hunk 2]
```diff
@@ -5,18 +5,39 @@
...
   width: 100%;
 }
+
+.md-menu-ripple {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+}
```
Comment (hansl) [inline src/lib/menu/menu.scss]: EOL.

File: src/lib/menu/menu-item.html (hunk 1 above)
Comment (kara) [inline src/lib/menu/menu-item.html]: Done

File: src/lib/menu/menu.scss (hunk 2 above)
Comment (kara) [inline src/lib/menu/menu.scss]: Done

**Comment by jelbourn** [comment]
LGTM

**Comment by kara** [comment]
I think the animation timing messed up e2es. Fixing.

**Comment by kara** [comment]
@hansl @jelbourn E2Es should be passing now, but unable to get SauceLabs connected. Restarted the build several times, but no joy. Ideas?

**Pull request merged**